### DISCUSSION TITLE
Does Resilience4j-spring-boot3 supports dynamic properties reload out of the box?

### DISCUSSION THREAD

**Discussion by RaghvendraGarg** [resilience4j/resilience4j#2180] [Q&A]
Hi Team,

Does R4j spring boot3 modules support dynamic configuration reload during runtime for circuit breaker? there is no reslience4j-spring-cloud3 module so I am assuming the support for dynamic configuration is built in?

Thanks,
R

**Comment by breun** [comment]
It feels like Spring Cloud refresh support was never implemented for use with Spring Boot 3. I think that would need a `resilience4j-spring-cloud3` module indeed.

**Reply by RobWin** [reply]
Hi, yes indeed. There is a PR, but I didn't have time to review it. Help appreciated
